models/repo.go

```diff
@@ -666,6 +668,41 @@ func (repo *Repository) CanEnableEditor() bool {
 	return !repo.IsMirror
 }
 
+// Find all Dependencies an issue is blocked by
+func (repo *Repository) BlockedByDependencies(issueID int64)  (_ []*Issue, err error) {
```

Comment:
Replace `_ []*Issue` with `issueDepsFull []*Issue` and remove `var issueDepsFull = make([]*Issue, 0)`.